gcloud/search/document.py

```diff
@@ -0,0 +1,303 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Define API Document."""
+
+import datetime
+
+import six
+
+from gcloud._helpers import UTC, _RFC3339_MICROS
```

Comment:
Can you rewrite as

```python
from gcloud._helpers import _RFC3339_MICROS
from gcloud._helpers import UTC
```

----

This is Google's internal style guide infecting me. Also it's a bit of "why start now" since we haven't done multiple imports on one line yet (`$ git grep 'import ' | grep ','`)